eclair-core/src/main/scala/fr/acinq/eclair/remote/EclairInternalsSerializer.scala

```diff
@@ -99,7 +99,7 @@ object EclairInternalsSerializer {
       ("pathFindingExperimentConf" | pathFindingExperimentConfCodec) ::
       ("balanceEstimateHalfLife" | finiteDurationCodec)).as[RouterConf]
 
-  val overrideFeaturesListCodec: Codec[List[(PublicKey, Features[Feature])]] = listOfN(uint16, publicKey ~ variableSizeBytes(uint16, lengthPrefixedFeaturesCodec))
+  val overrideFeaturesListCodec: Codec[List[(PublicKey, Features[Feature])]] = listOfN(uint16, publicKey ~ lengthPrefixedFeaturesCodec)
```

Comment:
This is a backward-incompatible change: front and back need to be updated simultaneously.